habitat-baselines/habitat_baselines/config/habitat_baselines/rl/policy/hierarchical_tp_noop_onav.yaml

```diff
@@ -0,0 +1,72 @@
+
+name: "HierarchicalPolicy"
+obs_transforms:
+  add_virtual_keys:
+    virtual_keys:
+      "goal_to_agent_gps_compass": 2
+hierarchical_policy:
+  high_level_policy:
+    name: "FixedHighLevelPolicy"
+    add_arm_rest: True
+  
+  defined_skills:
+    open_cab:
+      skill_name: "NoopSkillPolicy"
+      max_skill_steps: 1
+      apply_postconds: True
+
+    open_fridge:
+      skill_name: "NoopSkillPolicy"
+      max_skill_steps: 1
+      apply_postconds: True
+
+    close_cab:
+      skill_name: "NoopSkillPolicy"
+      obs_skill_inputs: ["obj_start_sensor"]
+      max_skill_steps: 1
+
+    close_fridge:
+      skill_name: "NoopSkillPolicy"
+      obs_skill_inputs: ["obj_start_sensor"]
+      max_skill_steps: 1
+      apply_postconds: True
+
+    pick:
+      skill_name: "NoopSkillPolicy"
+      obs_skill_inputs: ["obj_start_sensor"]
+      max_skill_steps: 1
+      apply_postconds: True
+
+    place:
+      skill_name: "NoopSkillPolicy"
+      obs_skill_inputs: ["obj_goal_sensor"]
+      max_skill_steps: 1
+      apply_postconds: True
+
+    nav_to_obj:
```

Comment:
Looks like the only difference between hierarchical_srl_onav.yaml and hierarchical_tp_srl.yaml is `nav_to_obj` being replaced with `gt_nav`. I think this is kind of a waste here, maybe both could inherit from a base configuration. Also, the name does not really reflect that this is the only difference, I think the name should change (rename hierarchical_srl_onav.yaml to hierarchical_tp_srl_onav.yaml).